code/modules/paperwork/paper.dm

```diff
@@ -623,6 +623,21 @@
 	name = "DJ Listening Outpost"
 	info = "<B>Welcome new owner!</B><BR><BR>You have purchased the latest in listening equipment. The telecommunication setup we created is the best in listening to common and private radio fequencies. Here is a step by step guide to start listening in on those saucy radio channels:<br><ol><li>Equip yourself with a multi-tool</li><li>Use the multitool on each machine, that is the broadcaster, receiver and the relay.</li><li>Turn all the machines on, it has already been configured for you to listen on.</li></ol> Simple as that. Now to listen to the private channels, you'll have to configure the intercoms, located on the front desk. Here is a list of frequencies for you to listen on.<br><ul><li>145.7 - Common Channel</li><li>144.7 - Private AI Channel</li><li>135.9 - Security Channel</li><li>135.7 - Engineering Channel</li><li>135.5 - Medical Channel</li><li>135.3 - Command Channel</li><li>135.1 - Science Channel</li><li>134.9 - Mining Channel</li><li>134.7 - Cargo Channel</li>"
 
+/obj/item/paper/ussp_tele
+	name = "Report - Bluespace Translocation Apparatus"
+	info = "Despite a lot of headaches and delays, the BTA is finally in a functional enough state to demonstrate to the Committee, although be warned that it has multiple limitations:<br>\
+	<br>\
+	* The highly sensitive electronics are affected by a translocation event, which will throw off any acquired locks.\
+	The station must be recalibrated after every use to avoid the possibility of being translocated to a random point in space.\
+	<br>\
+	* Currently the system is only able to function with the help of locator beacons, we have been unable to safely generate arbitrary targeting coordinates.\
+	<br>\
+	* We cannot retrieve targets remotely, even with the aid of a locator beacon, energy consumption is too high to maintain a remote portal.<br>\
+	<br>\
+	Doctors Danielius and Hoang both believe that the first point can be worked out with some relatively minor technological improvements.\
+	Arbitrary targeting is technically possible but will remain infeasible for decades. Remote target retrieval will require an entirely different technique, \
+	although accounts of Syndicate contractor operations suggests that they possess a functioning apparatus capable of such a feat."
```

Comment:
You can use heredoc syntax to avoid having to add line continuation slashes to every line.
```suggestion
	info = {"Despite a lot of headaches and delays, the BTA is finally in a functional enough state to demonstrate to the Committee, although be warned that it has multiple limitations:<br>
	<br>
	* The highly sensitive electronics are affected by a translocation event, which will throw off any acquired locks.
	The station must be recalibrated after every use to avoid the possibility of being translocated to a random point in space.
	<br>
	* Currently the system is only able to function with the help of locator beacons, we have been unable to safely generate arbitrary targeting coordinates.
	<br>
	* We cannot retrieve targets remotely, even with the aid of a locator beacon, energy consumption is too high to maintain a remote portal.<br>
	<br>
	Doctors Danielius and Hoang both believe that the first point can be worked out with some relatively minor technological improvements.
	Arbitrary targeting is technically possible but will remain infeasible for decades. Remote target retrieval will require an entirely different technique,
	although accounts of Syndicate contractor operations suggests that they possess a functioning apparatus capable of such a feat."}
```

Reply:
Done.